tasks/setup.yml

```diff
@@ -27,8 +27,8 @@
     - "{{ django_stack_media_root }}"
     - "{{ django_stack_logdir }}"
     - "/var/log/{{ django_stack_app_name }}"
-    - "/var/log/celery"
-    - "/var/run/celery"
+    - "{{ '/var/log/celery' if django_stack_celery_worker else [] }}"
+    - "{{ '/var/run/celery' if django_stack_celery_worker else [] }}"
```

Comment:
Hmm, what's the effect of `[]` as an item in a with_items loop? Can we not use `omit` here? The `[]` strategy I've only used with `with_flattened`.

Reply:
run the playbook and you shall see the 🌈 ! 

I didn't try `omit` to be honest, it might work -- i dunno but `[]` certainly seemed to work. It only looped on the three items up top. I was running the playbook in `-C` mode before so I was able to run it multiple times until I saw the change report for the `celery` items go away